torchvision/ops/boxes.py

```diff
@@ -69,10 +69,11 @@ def batched_nms(
         _log_api_usage_once(batched_nms)
     # Benchmarks that drove the following thresholds are at
     # https://github.com/pytorch/vision/issues/1311#issuecomment-781329339
-    if boxes.numel() > (4000 if boxes.device.type == "cpu" else 20000) and not torchvision._is_tracing():
-        return _batched_nms_vanilla(boxes, scores, idxs, iou_threshold)
-    else:
-        return _batched_nms_coordinate_trick(boxes, scores, idxs, iou_threshold)
+    return _batched_nms_vanilla(boxes, scores, idxs, iou_threshold)
```

Comment:
Can we do what Angela suggested [here](https://github.com/pytorch/pytorch/issues/138026#issuecomment-2415381134)?